roles/pulp-resource-manager/templates/pulp-resource-manager.service.j2

```diff
@@ -3,10 +3,6 @@ Description=Pulp Resource Manager
 After=network-online.target
 Wants=network-online.target
 
-# This service will break if left running while PostgreSQL restarts.
-BindsTo=postgresql.service
-After=postgresql.service
```

Comment:
This systemd unit file is already a template. We could just make the service name a variable.

I agree with not depending on postgres being a systemd unit, but I think we should add a comment about it.

Reply:
My goal is to not have users have Pulp services depend on any other services other than `network-online.target`. I don't think we want to affect Pulp services when postgreSQL is on the same system in the same way we don't want to affect Pulp services when PostgreSQL is on a different system.

The reason this was here to start with was to handle cases where postgreSQL restarts. The issue is that what about Redis? What about when the OOM killer kills a process? What about when the network can't route to postgreSQL for 30 seconds and then resolves? We can resolve all of these cases by making pulp services `Restart=true`. With Pulp auto-restarting it no longer benefits from a systemd dependency since the services are now able to start/run independently.

This work could also be moved to its own ticket and probably be its own bug. What do you think about me moving this work to its own PR and filing an issue about Pulp service not auto-relaunching?

What do you think about this problem statement ^?

Reply:
It would be a good idea to be a separate ticket, but I do not think it is a strictly necessary (use of your time.)

I also agree with `Restart=true` instead of a templated dependency (I was playing devil's advocate). You just provided additional reasons for me to agree with it.

However, I think we should put a comment about `Restart=true`. Most SysAdmins won't look through the history of the systemd unit file for rationale before they customize it; they will just look at the current version on disk.

Reply:
I see what you mean about leaving a comment so admins understand its purpose. I'll add that now and push.

I'm going to not make the issue and also leave the changes in this PR because until the installer gets a towncrier changelog having a Redmine issue isn't that valuable since the change is needed along with this work anyway.